src/runtime/knossos-kernel.cuh

```diff
@@ -17,8 +17,8 @@ template<typename scalar_t> using tensor_accessor_2 =
 
 template <typename scalar_t, typename F>
 __global__ void map_kernel_1(
-    const tensor_accessor_1<scalar_t> input,
     tensor_accessor_1<scalar_t> output,
+    const tensor_accessor_1<scalar_t> input,
```

Comment:
Reordering the parameters so that the output parameter comes first, for consistency with `map2`. When the function was called `revmap` it made a bit more sense for the parameters to be in opposite orders, which is what the PyTorch tutorial does.